action.yml

```diff
@@ -21,6 +21,10 @@ inputs:
     description: Path to report JSON file
     required: true
     default: ./d2l-test-report.json
+  inject-github-context:
+    description: Change injection of GitHub Actions context at report submission time
+    required: true
+    default: auto
```

Comment:
This will allow better control over the injection of GitHub Actions context in the future. The allowed options do the following.
* `auto` - will inject context if it's missing from the report an emit a warning.
* `force` - will always override the context information at submission time
* `off` - if missing or incomplete context is in the report it will fail to submit
The main use case I'm thinking of is if you generate a report on 1 runner and then try to upload it in another job. We probably want info from the actual test run and not the github context at the time of submission. I'm working towards adding github context injection at report generation time but want the option to override it if we have a scenario we know is bad/wrong.